viewer/db.js

```diff
@@ -1730,61 +1750,118 @@ Db.getIndices = async (startTime, stopTime, bounding, rotateIndex) => {
       }
       if (index.startsWith('sessions2-')) { // sessions2 might not have prefix
         index = index.substring(10);
+      } else if (internals.queryExtraIndicesRegex.some(re => re.test(index))) {
+        // extra user-specified indexes from the queryExtraIndices don't have the prefix
+        isQueryExtraIndex = true;
       } else {
         index = index.substring(internals.prefix.length + 10);
       }
+
       let year; let month; let day = 0; let hour = 0; let len;
+      let queryExtraIndexTimeMatched = false; let queryExtraIndexTimeMatch;
 
-      if (+index[0] >= 6) {
-        year = 1900 + (+index[0]) * 10 + (+index[1]);
-      } else {
-        year = 2000 + (+index[0]) * 10 + (+index[1]);
-      }
+      if (isQueryExtraIndex) {
```

Comment:
With the default `arkime_sessions3-*`-style indexes we can just jump right down to the bit where we extract the year, month, day, etc. from the index name because we know exactly how long they are (which was handled above, with the `index.startsWith` and `index.endsWith` checks, etc., right around the `queryExtraIndicesRegex` checks). However, with these user-defined ones we really don't exactly how they'll be prefixed. We're going to make the assumption though that they'll use the same style of suffixes, so we're going to do some regex checks to determine if it's daily, hourly, weekly, etc. If they are, then we'll set the temporary `index` variable to the matching date/time part of the index name, so the same logic can be used below to pull the date/time values out.